Kumi.Game/Online/API/Accounts/APIAccount.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Newtonsoft.Json;
+
+namespace Kumi.Game.Online.API.Users;
+
+public class APIAccount
+{
+    /// <summary>
+    /// A special ID used to represent the system users, or any other user that is not logged in.
+    /// </summary>
+    public const int DEFAULT_SYSTEM_ID = 0;
```

Comment:
why not -1?

Reply:
IDs are generally unsigned, and there'll never be an account with the ID of 0.